src/Extensions/Diagnostics.HealthChecks/Internal/HttpHealthCheckParameters.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Net;
+using System.Net.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	internal class HttpHealthCheckParameters
+	{
+		public string EndpointName { get; }
```

Comment:
are these expected to be unique?  Do we have any other identifier?

Reply:
This is the name of the endpoint for current service, they should be unique. I think server creation will fail if you will try to create two endpoints with the same name.
Why would we need other identifiers?